0074-mobile-poc-modeled-coverage-rewards.md

```diff
@@ -20,12 +20,14 @@ Combined with other parameters like uptime (Heartbeats) and backhaul (Speed Test
 
 Building the MOBILE Network began with the Genesis Period, during which 5G Hotspot owners with Radios had to send just one Heartbeat in 24 hours to prove they were online. This was a necessary starting point to kickstart the deployments. However, to be successful, any network must be reliable, always available, and meet the expectations of its users. Using Modeled Coverage is an important step to measure the quality of coverage beyond self-reported data. In the future, it will be complemented and cross-checked by the data provided by Mobile Mappers.
 
-Additionally, Modeled Coverage lays a foundation to introduce more data sources like zoning and population density to measure the usefulness of coverage and motivate deployments in places where it matters the most. The addition of the location-based incentive points will also be possible once the Modeled Coverage is introduced in the Mobile Rewards calculations.
+This HIP lays the groundwork for rewarding Helium Mobile Hotspot operators based on coverage vs. just existence (status quo) Specifically:
+1. Proposes a framework to count coverage using res 12 hexes;
+2. Proposes a framework to start incorporating external data sources for calculating coverage, starting with the Obstruction Data Oracle. We expect this to be the first in a series of HIPs that will incorporate other data sources, such as zoning and population density, feedback from carrier members of the MOBILE DAO, network users and mappers, etc.
 
 
-# Stakeholders
+# Stakeholders & Deployment Impact
```

Comment:
Can we expand this a bit? Deployment impact should include what changes folks will see.
 For example:
- They will see changes to the Mobile explorer (new)
- They will see changes to rewards. You will need to update your firmware (if that's what they do) to comply with the change.
- They will see changes to how indoor coverage is rewarded 
- They will continue to see HIPs that support the mobile POC roadmap - which you can find here. 

Altogether, need to be explicit with folks about deployment impacts (what is changing).